whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/DomainAuthSpec.groovy

```diff
@@ -3,8 +3,9 @@ package net.ripe.db.whois.spec.update
 
 import net.ripe.db.whois.spec.BaseQueryUpdateSpec
 import net.ripe.db.whois.spec.domain.AckResponse
+import org.junit.jupiter.api.Tag
 
-@org.junit.jupiter.api.Tag("IntegrationTest")
+@Tag("IntegrationTest")
```

Comment:
The reason there is fully qualified class name is because we have a Tag class of our own. But I guess since this is not suing that so nice to use the short one

Reply:
Unless there are clashes I see no reason to litter the code with fully qualified class imports..